components/style/properties/longhand/svg.mako.rs

```diff
@@ -105,10 +105,13 @@ ${helpers.single_keyword("mask-repeat",
 
 <%helpers:vector_longhand name="mask-position-x" products="gecko" animatable="True" extra_prefixes="webkit"
                           spec="https://drafts.fxtf.org/css-masking/#propdef-mask-position">
-    use std::fmt;
-    use style_traits::ToCss;
-    use values::HasViewportPercentage;
-    use values::specified::position::HorizontalPosition;
+    use properties::animated_properties::{Interpolate, RepeatableListInterpolate};
+    use properties::longhands::mask_position_x::computed_value::T as MaskPositionX;
+    pub use properties::longhands::background_position_x::single_value::get_initial_value;
+    pub use properties::longhands::background_position_x::single_value::get_initial_position_value;
+    pub use properties::longhands::background_position_x::single_value::get_initial_specified_value;
+    pub use properties::longhands::background_position_x::single_value::parse;
+    pub use properties::longhands::background_position_x::single_value::SpecifiedValue;
 
     pub mod computed_value {
```

Comment:
Actually `background_position_{x,y}` also have interpolate implementations. But their `single_value::computed_value::T`s are pre defined type(VerticalPosition and HorizontalPosition), and their implementations are stored here: https://dxr.mozilla.org/servo/source/components/style/properties/helpers/animated_properties.mako.rs#698-716

Also you will need to add interpolate implementations for `computed_value::T` (it's **different** from `single_value::computed_value::T`, the computed_value module you write in the helper is single_value one. And main computed_value is generating by helper. Because it's a vector_longhand)
Like background ones: https://dxr.mozilla.org/servo/source/components/style/properties/helpers/animated_properties.mako.rs#716-730